service/src/main/java/bio/terra/workspace/db/FolderDao.java

```diff
@@ -48,8 +50,16 @@ public class FolderDao {
             Objects.requireNonNull(rs.getString("display_name")),
             rs.getString("description"),
             parentFolderId,
-            DbSerDes.jsonToProperties(Objects.requireNonNull(rs.getString("properties"))));
+            DbSerDes.jsonToProperties(Objects.requireNonNull(rs.getString("properties"))),
+            rs.getString("created_by_email"),
+            OffsetDateTime.ofInstant(
```

Comment:
Hmm... why doesn't Folder use Instant? I think it is what we use everywhere else.
Not for this PR. Seems like we should be consistent. Maybe the right answer is OffsetDateTime instead of Instant.

Reply:
i'm using OffsetDateTime because  swagger ui auto-generate date-time type to `OffsetDateTime`.

Reply:
I made [PF-2272](https://broadworkbench.atlassian.net/browse/PF-2272) to track this.